source/ftrack_connect_pipeline_qt/ui/asset_manager/base/__init__.py

```diff
@@ -52,17 +53,25 @@ def snapshot_assets(self):
     @property
     def is_assembler(self):
         '''Return asset list widget'''
-        return self._client.is_assembler
+        return self.client.is_assembler
```

Comment:
Just to clarify, why are we changing client from private to public? is it used outside?

Reply:
I'll change back if it is more clear, just wanted to access client property the same way throughout the code so we can change the getter later on if needed